index.html

```diff
@@ -4,10 +4,10 @@
     <meta charset="UTF-8">
     <title>Serious game for kids</title>
 </head>
-<body>
-    <div class="wrapper">
-        <section id="top-bar" class="navbar navbar-light bg-faded" style="border-bottom: 1px solid #ccc;"></section>
-        <div class="container-fluid">
+<body style="padding-top: 70px">
```

Comment:
I know I've not been all that consistent in this either, but could you put this inline css into a css file? Perhaps we can rename the css file we have into something more general like `default.css`, instead of `overlay.css`.